.travis.yml

```diff
@@ -23,4 +23,4 @@ before_script:
   - psql -c 'create database travisci;' -U postgres
   - python manage.py migrate
 script:
-  - python manage.py test
\ No newline at end of file
+  - python manage.py test
```

Comment:
Should this be changed to `coverage run mange.py test`?

Reply:
If you see in the first PR, I added (coverage run manage.py test & coverage report), but travis failed on its checks.

Reply:
Probably because of missing dependencies in `requirements.txt`.